spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -49,8 +49,9 @@
         sign_in_before_2fa
 
         properties = {
-          context: 'authentication',
           success?: false,
+          confirmation_for_phone_change: false,
+          context: 'authentication',
```

Comment:
seeing this string in the specs makes me think that my earlier comment (re: making this a constant) may be correct. I can imagine a typo in the word creating a confusingly failing spec.